src/devtools/pachdev/actions.go

```diff
@@ -0,0 +1,571 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"slices"
+	"strings"
+
+	"github.com/mikefarah/yq/v4/pkg/yqlib"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/protobuf/types/known/emptypb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cmd"
+	"sigs.k8s.io/kind/pkg/log"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/client"
```

Comment:
This is the most up-to-date client in our repo.  src/client has diverged which is bad 😂